content/python/concepts/classes/classes.md

```diff
@@ -6,51 +6,58 @@ Subjects:
   - 'Data Science'
   - 'Web Development'
 Tags:
+  - 'Attributes'
   - 'Classes'
+  - 'Methods'
   - 'Objects'
   - 'OOP'
 CatalogContent:
   - 'learn-python-3'
   - 'paths/computer-science'
 ---
 
-**Classes** are templates used to define the properties and methods of objects in code. They can describe the kinds of information that the class holds, and also how a programmer interacts with that data.
+**Classes** are templates used to define the properties and methods of objects in code. They can describe the kinds of data that the class holds, and also how a programmer interacts with that data.
 
 The usage of classes are a key element of [object-oriented programming](https://www.codecademy.com/resources/docs/general/object-oriented-programming) (OOP) where "class instances" are created to implement design principles such as [inheritance](https://www.codecademy.com/resources/docs/general/inheritance) and [encapsulation](https://www.codecademy.com/resources/docs/general/encapsulation).
 
 ## Creating a Class
 
-In Python, classes are defined using the `class` keyword.
-
-```py
-class Animal:
-  def __init__(self, name, leg_count):
-    self.name = name
-    self.leg_count = leg_count
-```
-
-Class definitions cannot be empty, so the `pass` statement can be used as a placeholder to avoid errors.
-
 ```py
 class Home:
-  pass
+  # Class body starts here
 ```
 
-## Creating Instances of a Class
+In Python, classes are defined using the [`class`](https://www.codecademy.com/resources/docs/python/keywords/class) keyword. Within the code of the class, we can call its methods and access its variables. We do this by using `self` followed by a period `.` and then followed by the method or variable.
 
-Objects can be created from classes. These objects are called instances of a class, and when we create an instance, that is known as instantiating a class.
+### Class Instances
 
-To create an instance of a class, we can set a variable equal to the class name followed by parentheses `()`.
+Objects can be created or instantiated, from classes. These objects are known as class instances. To create an instance of a class, we can set a variable equal to the class name followed by parentheses `()`:
 
 ```py
 my_home = Home()
 ```
 
 Here, the instance name is `my_home`, and the class is `Home`.
 
-## Class Methods
+### Attributes
+
+Class attributes are variables that are defined outside of all methods and have the same value for every instance of the class.
+
+```py
+class Bird:
+  # Class attribute
+  leg_count = 2
+
+parakeet = Bird()
+parrot = Bird()
+
+print(parakeet.leg_count) # Output: 2
+print(parrot.leg_count)   # Output: 2
+```
```

Comment:
It might be a good idea to have an example where the Class attribute is changed, and we show it changing for the various instances.